website/docs/prysm-usage/checkpoint-sync.md

```diff
@@ -4,68 +4,207 @@ title: Sync from a checkpoint
 sidebar_label: Sync from a checkpoint
 ---
 
+import CheckpointSyncPresent from '@site/static/img/checkpoint-sync-present.png';
+import CheckpointSyncAbsent from '@site/static/img/checkpoint-sync-absent.png';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 :::caution
 
 **This feature is currently in public preview** and may change significantly as we receive feedback from users like you. Join our [Discord server](https://discord.gg/prysmaticlabs) to share your feedback.
 
 :::
 
-Prysm provides the ability to sync from a finalized checkpoint, as an alternative to replaying all history starting from Genesis. Checkpoint Sync is significantly faster than Genesis Sync, and is considered more secure thanks to the protections against long-range attacks afforded by [Weak Subjectivity](https://blog.ethereum.org/2014/11/25/proof-stake-learned-love-weak-subjectivity/).
+**Checkpoint sync** is a feature that significantly speeds up the initial sync between your beacon node and the Beacon Chain. With checkpoint sync configured, your beacon node will begin syncing from a recently finalized checkpoint instead of syncing from genesis. This can make installations, validator migrations, recoveries, and testnet deployments *way* faster.
```

Comment:
I think it would be good to emphasize on trust here that you trust some source and the safest way is always still to sync from genesis.

Reply:
It is not strictly safer to sync from genesis, due to long range attacks - this is the idea behind weak subjectivity:
https://blog.ethereum.org/2014/11/25/proof-stake-learned-love-weak-subjectivity/

However the checkpoint block and root should be verified against a trusted source. This is why I asked Mick to include the instructions to verify roots once the node has been started from checkpoint sync. It could be good to emphasize at the beginning of the document that users should follow that verification procedure. Putting a fine point on it *I think it is more important to emphasize the importance of the verification procedure than to emphasize the trusted nature of the node providing the block+state*. You do not need to trust the other node, so long as you have a trusted source for confirming hashes.

Reply:
@kasey made an interesting point that the source of the state technically doesn't actually need to be trusted - but that the state you use should instead be verified using a trusted blockchain explorer. I'll add a note with this nuance...